RDF_Tests/ToTTLTests.cs

```diff
@@ -384,6 +388,58 @@ public static void NurbsCurve_ControlPoints()
             Assert.IsTTLParsable(TTLGraph);
         }
 
-   
+        [Test]
+        public static void UriGuidBHoMObj()
+        {
+            Room room = new Room();
+            room.Perimeter = new Polyline() { ControlPoints = new List<Point>() { new Point(), new Point() { X = 5 }, new Point() { X = 99 } } };
+            room.Location = new Point();
+            room.Name = "first room object";
+
+            CSharpGraph cSharpGraph = Compute.CSharpGraph(new List<object>() { room }, m_ontologySettings);
+            string TTLGraph = cSharpGraph.ToTTL();
+
+            Assert.IsTrue(TTLGraph.Contains(room.BHoM_Guid.ToString()));
```

Comment:
This check should be replaced with:
```cs
string roomUrl = Flurl.Url.Combine(cSharpGraph.OntologySettings.ABoxSettings.IndividualsBaseAddress, room.BHoM_Guid.ToString());
Assert.IsTrue(TTLGraph.Contains(roomUrl));
```
(will add a commit for you)

Reply:
Done in https://github.com/BHoM/RDF_Prototypes/pull/68/commits/c0f70ecdd9093e071ee5bc2837888cd65306d24e